libraries/corretto_helpers.rb

```diff
@@ -3,36 +3,52 @@ module Cookbook
     module CorrettoHelpers
       def default_corretto_url(version)
         if version.to_s == '8'
-          'https://corretto.aws/downloads/latest/amazon-corretto-8-x64-linux-jdk.tar.gz'
-        else
-          'https://corretto.aws/downloads/latest/amazon-corretto-11-x64-linux-jdk.tar.gz'
+          if node['kernel']['machine'].match?('x86_64')
+            'https://corretto.aws/downloads/latest/amazon-corretto-8-x64-linux-jdk.tar.gz'
+          elsif node['kernel']['machine'].match?('aarch64')
+            'https://corretto.aws/downloads/latest/amazon-corretto-8-aarch64-linux-jdk.tar.gz'
+          end
```

Comment:
Instead of doing it this way, it might make sense to make a private method like the following:
```ruby
def correto_arch
  node['kernel']['machine'].match?('x86_64') ? 'x86' : 'aarch64'
end
```
And then you can just use ``"https://corretto.aws/downloads/latest/amazon-corretto-8-#{correto_arch}-linux-jdk.tar.gz"`` instead.

Reply:
thanks for the suggestion!